Source/Archiver.h

```diff
@@ -51,7 +51,7 @@ SOFTWARE
 #include <View.h>
 #include <Window.h>
 
-#include <be/add-ons/tracker/TrackerAddon.h>
+#include </boot/system/develop/headers/os/add-ons/tracker/TrackerAddOn.h>
```

Comment:
mm but it didn't compiled 
"boot/home/Archiver/Source/Archiver.h:54: /add-ons/tracker/TrackerAddOn.h: No such file or directory
/boot/system/develop/etc/makefile-engine:291: recipe for target 'objects.x86-gcc2-release/Archiver.o' failed
make: *** [objects.x86-gcc2-release/Archiver.o] Error 1
"